src/routers/apiRouter.js

```diff
@@ -2,20 +2,36 @@ const userController = require('../controllers/user')
 const placeController = require('../controllers/place')
 const eventController = require('../controllers/event')
 const productController = require('../controllers/product')
-const validateJWT = require('../middleware/validateJWT')
-const validateHeaderToken = require('../middleware/validateHeaderToken')
+
+// const validateJWT = require('../middleware/validateJWT')
+// const validateHeaderToken = require('../middleware/validateHeaderToken')
+// const validateUser = require('../middleware/validateUser')
+// const permissions = require('../middleware/permissions')
+// const fieldPermissions = require('../middleware/fieldPermissions')
+
+// const api = require('../constants/apiPermissions')
 
 const router = require('express').Router()
 
 // user routes
 router.route('/users')
   .get(userController.getAll)
-  .post(userController.create)
+  .post(
+    // validateJWT(),
+    // validateUser(),
+    // validateHeaderToken,
+    // fieldPermissions(api.User.fields),
+    userController.create
+  )
+
 router.route('/users/:id')
   .get(userController.getById)
   .put(
-    validateJWT(),
-    validateHeaderToken,
+    // validateJWT(),
```

Comment:
dito

Reply:
it was kinda half-done already, now it's still half-done but differently.